src/api/users/src/routes/users.js

```diff
@@ -111,6 +111,8 @@ router.post(
         next(createError(409, `user with id ${id} already exists.`));
       } else {
         const user = new User(body);
+        // no user can be created as an admin by default
+        user.isAdmin = false;
```

Comment:
I would move this code into a new middleware function that happens just before the route's logic gets hit.  Maybe call it `validateUserRights()` and it can simply set `user.isAdmin` to `false`.

The logic in the route should be focused on talking to the db and returning HTTP results, nothing more.

Reply:
`validateUserRights` created.